pkg/apis/minio.min.io/v2/types.go

```diff
@@ -196,10 +221,16 @@ type TenantSpec struct {
 	Readiness *corev1.Probe `json:"readiness,omitempty"`
 
 	// *Optional* +
+	// *Deprecated in Operator v4.3.2* +
 	//
 	// S3 related features can be disabled or enabled such as `bucketDNS` etc.
 	S3 *S3Features `json:"s3,omitempty"`
 	// *Optional* +
+	// *Deprecated in Operator v4.3.2* +
+	//
+	// S3 related features can be disabled or enabled such as `bucketDNS` etc.
+	Features *Features `json:"features,omitempty"`
+	// *Optional* +
```

Comment:
this is not deprecated right? this is a new feature

Reply:
nice catch